src/parser.yy

```diff
@@ -710,12 +729,12 @@ call:
         |       CALL "(" vargs ")"           { $$ = driver.ctx.make_node<ast::Call>($1, std::move($3), @$); }
         |       CALL_BUILTIN  "(" ")"        { $$ = driver.ctx.make_node<ast::Call>($1, @$); }
         |       CALL_BUILTIN "(" vargs ")"   { $$ = driver.ctx.make_node<ast::Call>($1, std::move($3), @$); }
-        |       IDENT "(" ")"                { error(@1, "Unknown function: " + $1); YYERROR;  }
-        |       IDENT "(" vargs ")"          { error(@1, "Unknown function: " + $1); YYERROR;  }
         |       BUILTIN "(" ")"              { error(@1, "Unknown function: " + $1); YYERROR;  }
         |       BUILTIN "(" vargs ")"        { error(@1, "Unknown function: " + $1); YYERROR;  }
         |       STACK_MODE "(" ")"           { error(@1, "Unknown function: " + $1); YYERROR;  }
         |       STACK_MODE "(" vargs ")"     { error(@1, "Unknown function: " + $1); YYERROR;  }
+        |       IDENT "(" ")"                { $$ = driver.ctx.make_node<ast::Call>($1, @$); }
+        |       IDENT "(" vargs ")"          { $$ = driver.ctx.make_node<ast::Call>($1, std::move($3), @$); }
```

Comment:
This reminds me: I had a change to strip the list of builtins and "calls" out of the lexer, which I think can now be done without issue. Did you want to do that? (Once you add macros the distinction being made between these things is now not useful, so we can decompose everything into either `Call` or `Builtin`)

Reply:
> I had a change to strip the list of builtins and "calls" out of the lexer, which I think can now be done without issue. Did you want to do that? 

Sure, that sounds like a fun cleanup.

Reply:
https://github.com/bpftrace/bpftrace/pull/4043